### DISCUSSION TITLE
Listener log

### DISCUSSION THREAD

**Discussion by infusopharm** [chuot/rdio-scanner#246] [Ideas]
Would be great if able to view the log of listener usage to show date and time.

**Comment by chuot** [comment]
The are already logged with date and time.

If you are running Rdio Scanner as a service, you can grep the lines using `journalctl` on Linux or filter them with the `event viewer` on Windows.

**Comment by infusopharm** [comment]
Thanks for the response. I will try to figure out how to run the windows
command prompts as a service.

Thanks

**Reply by chuot** [reply]
Look at page 7 of the PDF document that comes with Rdio Scanner.

**Reply by infusopharm** [reply]
In event viewer I can see all the new calls listed but no record of users logging in or out

**Comment by infusopharm** [comment]
Played around with it last night. Was seeing the log files being imported
in but was not seeing any logs about listeners. I will play around with it
a little more.
Thanks

**Comment by chuot** [comment]
@infusopharm Just uploaded v6.6.0, please try with it. I fixed an issue in regards to logs not being updated and listeners count should be there too.
